lms/djangoapps/course_home_api/progress/v1/tests/test_views.py

```diff
@@ -0,0 +1,68 @@
+"""
+Tests for Progress Tab API in the Course Home API
+"""
+
+from datetime import datetime
+import ddt
+
+from django.urls import reverse
+
+from course_modes.models import CourseMode
+from lms.djangoapps.course_home_api.tests.utils import BaseCourseHomeTests
+from lms.djangoapps.course_home_api.toggles import COURSE_HOME_MICROFRONTEND
+from openedx.features.content_type_gating.models import ContentTypeGatingConfig
+from openedx.core.djangoapps.user_api.preferences.api import set_user_preference
+from openedx.core.djangoapps.waffle_utils.testutils import override_waffle_flag
+from student.models import CourseEnrollment
+from student.tests.factories import UserFactory
```

Comment:
Doesn't look like this is used anywhere.

Reply:
i cleaned up all the imports so only the ones that are used are in!

Reply:
Ah noticed it in the test_masquerade. Thanks!